findimports.py

```diff
@@ -813,23 +814,16 @@ def packageGraph(self, packagelevel=None, externals_only=False):
         graph.modules = packages
         return graph
 
-    @staticmethod
-    def rmStrPrefix(string, prefixes):
-        """Remove prefix from a string and strip any leading dots"""
-        for prefix in prefixes:
-            if string.startswith(prefix):
-                return string[len(prefix):].lstrip('.')
-        return string
-
     def removePrefixes(self, prefixes):
         """Remove prefixes. Only applies 1st hit."""
+        reg_cmp = re.compile(r'^(({})\.)?'.format('|'.join(prefixes)))
```

Comment:
```suggestion
        reg_cmp = re.compile(r'^(({})\.)?'.format('|'.join(map(re.escape, prefixes))))
```

(or, alternatively, document in --help that --rmprefix accepts regular expressions, but I don't necessarily like that idea.)

Reply:
added escape.